nectar/src/config.rs

```diff
@@ -30,13 +30,10 @@ pub struct Bitcoind {
 }
 
 #[derive(Clone, Debug, Default, Deserialize, PartialEq, Serialize)]
-pub struct MaxSell {
+pub struct Max {
     #[serde(default)]
```

Comment:
Does this design still make sense? Looking at it I think I agree with @thomaseizinger, we should introduce the pairs in the config file.

Reply:
Do we really want to tackle this already? I added another comment above.

Reply:
Don't tackle it if you don't want to, it is up to you :)